pkgs/top-level/beam-packages.nix

```diff
@@ -93,7 +93,7 @@ in
   # appropriate Erlang/OTP version.
   packages = {
     erlang = self.packages.${self.latestVersion};
-
+    erlang_27 = self.packagesWith self.interpreters.erlang_27;
```

Comment:
I literally tried doing this today but `nix-build -A beam.packages.erlang_27` will fail due to LFE. I asked about this in Slack waiting to hear back.

Reply:
It's possible to reference previous erlang version for lfe until it isn't updated.

Reply:
Just updated `lfe` and seems to build fine.

Reply:
It might be less about building. I was looking through the LFE git repo today and it made mention of OTP < 26 doing something one way, OTP 26 doing it another way, and future OTP versions doing something else. I don't remember the details. The point being we should test the resulting LFE binaries to see if it actually works when built with 27.